build.gradle

```diff
@@ -47,15 +47,15 @@ configurations {
 }
 
 jib {
-	container {
-		jvmFlags = [
-				'-Dotel.exporter=jaeger',
-				'-Dotel.exporter.jaeger.endpoint=http://jaeger:14250',
-				'-Dotel.exporter.jaeger.service.name=rules',
-				'-Dapplication.name=rules',
-				'-javaagent:/opentelemetry-javaagent-all.jar'
-		]
-	}
+    container {
+        jvmFlags = [
+                '-Dotel.exporter=jaeger',
+                '-Dotel.exporter.jaeger.endpoint=http://jaeger:14250',
+                '-Dotel.exporter.jaeger.service.name=rules',
+                '-Dapplication.name=rules',
+                '-javaagent:/opentelemetry-javaagent-all.jar'
+        ]
+    }
 }
 
 jar{
```

Comment:
update the dependencies as per https://repo.maven.apache.org/maven2/com/techsophy/

Reply:
for lib-id generator and multitenancy mongo

Reply:
and verify that compilation works fine